applications/RANSApplication/python_scripts/formulations/utilities.py

```diff
@@ -0,0 +1,244 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+import KratosMultiphysics.RANSApplication as KratosRANS
+
+from KratosMultiphysics import IsDistributedRun
+from KratosMultiphysics import VariableUtils
+from KratosMultiphysics.kratos_utilities import CheckIfApplicationsAvailable
+
+from KratosMultiphysics.RANSApplication import RansVariableUtilities
+
+if (IsDistributedRun()
+        and CheckIfApplicationsAvailable("TrilinosApplication")):
```

Comment:
```suggestion
if (IsDistributedRun() and CheckIfApplicationsAvailable("TrilinosApplication")):
```

Reply:
Done locally, will push once you are done with the review :)